### ISSUE TITLE
Full screen aspect ratio change 

### ISSUE THREAD

**Issue by DarthSabbath** [AppleWin/AppleWin#1122]
Is there a way to change the aspect ratio of Fullscreen mode?

**Comment by tomcw** [comment]
Yes: from History.txt:

- Command line: Allow user to specify width & height (for full-screen); and allow separate x,y scaling in full-screen mode.
  - eg. for 4:3 aspect ratio on monitors that support it: -no-full-screen -fs-width=1600 -fs-height=1200

NB. Combine with `-no-full-screen` to start in Windowed mode. Without this it'll just default to full-screen.
NB. When switching to Windowed mode the default desktop resolution will be restored, and when switching back to full-screen mode this better resolution will again be used.

EG:
`applewin.exe -no-full-screen -fs-width=1600 -fs-height=1200`

btw, I see that `-fs-width` isn't documented in the AppleWin.chm, so I'll fix that.

**Comment by DarthSabbath** [comment]
still doesn't work sadly, its still stretching itself in fullscreen

**Comment by DarthSabbath** [comment]
nevermind, got it working if i use 1280 x 960

**Comment by tomcw** [comment]
btw, What's the native resolution of your monitor?

**Comment by DarthSabbath** [comment]
1920 x 1024

**Comment by DarthSabbath** [comment]
also another question, how do I get the mockingboard working for the ultima games?

**Comment by tomcw** [comment]
Can you try this new [AppleWin 1.30.12.0](https://github.com/AppleWin/AppleWin/releases/tag/v1.30.12.0) ?

NB. not using any (-fs-height/width) command line options will now only scale by the same amount for x and y.

**Comment by philgoetz** [comment]
I tried this with v1.30.11.0:
$ ./AppleWin.exe -no-full-screen -fs-width=1280 -fs-height=960 &

I got the regular, non-stretchable, usual-size (about 1550 x something) window.

**Comment by Rekrullurker** [comment]
I just wanted to say thank you. I originally came here to suggest that AppleWin allow setting the fullscreen resolution (which I didn't see in the GUI), and I learned that it's already implemented in the command line options. I have an old system and still use a 4:3 monitor for daily use, so a resolution of 1152x864 is just about perfect. The buttons are mostly off the right side of the screen, but can still be accessed, or I can use the function keys. :)

**Comment by tomcw** [comment]
Great - thanks for the feedback. I sometimes wonder how useful (and discoverable) these obscure command line switches are, so good to know you found them, and they were useful!

**Comment by Rekrullurker** [comment]
To be honest, when it comes to emulators, unless it's entirely command line based, like Mednafen, I tend to just look for the configuration options in the GUI. I usually think of the console as just being used for stuff like feeding it a game to launch, or whether to start in a window or fullscreen. It kind of catches me by surprise when a program has a GUI, but major configuration options are only available via the command line.

**Comment by tomcw** [comment]
>It kind of catches me by surprise when a program has a GUI, but major configuration options are only available via the command line.

Yeah, the configuration GUI needs an overhaul. It's being tracked in #853, but little progress recently.

**Comment by tomcw** [comment]
Closing old question.
